src/Dapr.Client/DaprClient.cs

```diff
@@ -984,9 +984,9 @@ public async Task<StateEntry<TValue>> GetStateEntryAsync<TValue>(string storeNam
         /// <param name="workflowOptions">The list of options that are potentially needed to start a workflow.</param>
         /// <param name="input">The input input for the given workflow.</param>
         /// <param name="cancellationToken">A <see cref="CancellationToken" /> that can be used to cancel the operation.</param>
-        /// <returns>A <see cref="Task"/> containing a <see cref="WorkflowReference"/></returns>
+        /// <returns>A <see cref="Task"/> containing a <see cref="StartWorkflowResponse"/></returns>
         [Obsolete("This API is currently not stable as it is in the Alpha stage. This attribute will be removed once it is stable.")]
-        public abstract Task<WorkflowReference> StartWorkflowAsync(
+        public abstract Task<StartWorkflowResponse> StartWorkflowAsync(
```

Comment:
What's the backing reason for this? I know this is a preview API, but generally, I'd like to make the landing softer for something that would be a breaking change. 

I've read the code below and understand that this isn't a super easy thing to adjust because the original `WorkflowReference` is sealed and thus cannot be extended.

In the end, I'm OK with a breaking change cause that's the point of an alpha API, I'd just like to know a bit more about why.

Reply:
For this release each of the various workflow methods were given their own unique type of response object. The reason for this is to allow for more relevant responses for each of the calls in the future.